core/src/test/scala/unit/kafka/server/AlterPartitionManagerTest.scala

```diff
@@ -552,24 +535,10 @@ class AlterPartitionManagerTest {
 }
 
 object AlterPartitionManagerTest {
-  def provideMetadataVersions(): JStream[MetadataVersion] = {
+  def provideLeaderRecoveryState(): JStream[Arguments] = {
```

Comment:
we can use `@EnumSource(classOf[LeaderRecoveryState])`, thus this method can remove